### ISSUE TITLE
[Bug]: [P1] Common accessibility issue - Missing headings in component pages

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ods-ios#573]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 Plus

### OS Version

17.1.1

### App version

v0.16.0

### Describe the issue

For each component, the "Variants" label is visually a section, but it's not defined as a header. It's useful for users navigating with Voice Over to reach quickly this section.

![image](https://github.com/Orange-OpenSource/ods-ios/assets/105045667/e1d36e03-22f1-4b4a-bd37-ac73ab56c5e2)

### Expected Behavior

Add `.accessibilityAddTraits(.isHeader)` to "Variants" label

**Comment by pylapp** [comment]
After having investigated accros the UI, @pya35 it seems this issue has been fixed with the commit 57b2923 (with #553) delivered with version 0.16.0 (https://github.com/Orange-OpenSource/ods-ios/releases/tag/0.16.0)

**Comment by pya35** [comment]
OK fixed
